### PR TITLE
Fix #49, refactor write functions to mesh

### LINKED ISSUE #49: Refactor functions that write output files to mesh instead of io

### PR DESCRIPTION
**Pull request by ezrayst**

### CODE REVIEW

File: include/mesh.h [hunk 1]
```diff
@@ -73,6 +85,9 @@ class Mesh {
   //! Map to store element id and vertices coordinates
   std::map<unsigned, Eigen::VectorXd> elementcoordinates_;
 
+  //! Map to store element id and volume of the element
+  std::map<unsigned, double> volumes_;
+
```
Comment (cw646) [inline include/mesh.h]: @kks32 didn't we decide not to store volumes?

Comment (ezrayst) [reply include/mesh.h]: @cw646, I was just thinking it is cleaner and more consistent this way.

Previously you just pass the whole map to `io` and now that we are doing it in `mesh` I can make it not to store and to have just one function to `compute` and to `write`. What do you think?

Comment (kks32) [reply include/mesh.h]: Material points will know it's volume, can we just iterate over points and print volumes?

**Review by kks32 — CHANGES_REQUESTED** [review]

**Review by kks32 — CHANGES_REQUESTED** [review]
Could you please rebase with the develop branch, as IO handling has changed.

**Review by kks32 — APPROVED** [review]

**Pull request merged**